pkgs/development/python-modules/pykdl/default.nix

```
@@ -1,9 +1,19 @@
-{ lib, stdenv, toPythonModule, cmake, orocos-kdl, python, sip }:
+{ lib, stdenv, toPythonModule, fetchpatch, cmake, orocos-kdl, python, sip }:
 
 toPythonModule (stdenv.mkDerivation {
   pname = "pykdl";
   inherit (orocos-kdl) version src;
 
+  patches = [
+    # Fix build with SIP 4.19.23+
```

```suggestion
    # Fix build with SIP 4.19.23+. Can be removed with version 1.4.1.
```
Not strictly required but I always think it is nice to have.

---

There's no indication that they plan to backport this to the 1.4 branch. I guess we can assume it will be part of 1.5.